docs/access-token.md

```diff
@@ -0,0 +1,61 @@
+# Access Token
+
+If you want to automate tasks with the Argo Server API or CLI, you will need an access token. 
+
+Firstly, create a role with minimal permissions. This example role for jenkins only permission to update and list workflows:
+
+```shell script
+kubectl create role jenkins --verb=list,update --resource=workflows.argoproj.io 
+```
+
+Create a service account for your service:
+
+```shell script
+kubectl create sa jenkins
+```
+
+Bind the service account to the role (in this case in the `argo` namespace):
+
+```shell script
+kubectl create rolebinding jenkins --role=jenkins --serviceaccount=argo:jenkins
+```
+
+You now need to get a token:
+
+```shell script
+SECRET=$(kubectl get secret  -o name | grep '^secret/jenkins-token-' | sed 's/secret\///')
```

Comment:
Need to decode the token.

A workable script:

```
ARGO_TOKEN=$(kubectl -n argo get secret `kubectl -n argo get sa jenkins -o=jsonpath='{.secrets[0].name}'` -o=jsonpath='{.data.token}' | base64 --decode)
```

Reply:
Nice - really good. Thank you.